ng-dev/pr/merge/merge-pull-request.ts

```diff
@@ -114,12 +121,22 @@ export async function mergePullRequest(prNumber: number, flags: PullRequestMerge
 async function createPullRequestMergeTool(flags: PullRequestMergeFlags) {
   try {
     const config = await getConfig();
+
     assertValidGithubConfig(config);
     assertValidPullRequestConfig(config);
+    assertValidCaretakerConfig(config);
+
+    let googleSyncConfig: GoogleSyncConfig | null = null;
+    if (config.caretaker.g3SyncConfigPath) {
```

Comment:
Can this be moved downstream in the process until its actually needed?

Reply:
Yes. I'll update.